### PR TITLE
March Knowledge Center Updates

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes

All feedback in [hack.md](https://hackmd.io/REYhSNQ9TuSYgCQhxOUPZA) is addressed/new content added
Reorganized FAQ page around emerging categories of question to make it easier to parse
Added mention/link to Dear Colleague letter on the homepage

## How to Test
FAQ Page in Test Environment: https://reitermb.github.io/Test/knowledge-center/faq.html
Homepage in Test Environment: https://reitermb.github.io/Test/knowledge-center/index.html

### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [x] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### CODE REVIEW

File: product-updates/knowledge-center/faq.html [hunk 1]
```diff
@@ -267,46 +267,97 @@ <h1>Frequently Asked Questions</h1>
...
+        <h3 style="font-size: 1.34rem; line height: 1.2; font-weight: 700;">Are there restrictions governing where TDP can be used?</h3>
+        <P>The TANF Data Portal can be used from any computer in the US or US Territories. There is no need to get any particular computer or IP Address “pre-approved” in order to use TDP.</p>
           <div style="min-height: 22px;"></div>
           <div style="min-height: 22px;"></div>
 
 
-        <h2 style="font-size: 1.34rem; line height: 1.2; font-weight: 700;">Are there restrictions governing where TDP can be used?</h2>
-        <P>The TANF Data Portal can be used from any computer in the US or US Territories. There is no need to get any particular computer or IP Address “pre-approved” in order to use TDP.</p>
+        <h3 style="font-size: 1.34rem; line height: 1.2; font-weight: 700;">Can multiple users on my team request access to TDP?</h3>
+        <P>Yes. We're not limiting the number of users who can be request access per TANF program. We recommend that a minimum of two users request access per team to ensure coverage due to leave or unforeseen circumstances.</p>
```
Comment (lfrohlich) [inline product-updates/knowledge-center/faq.html]:
```suggestion
        <P>Yes. We're not limiting the number of users who can request access per TANF program. We recommend that a minimum of two users request access per team to ensure coverage due to leave or unforeseen circumstances.</p>
```

File: product-updates/knowledge-center/index.html [hunk 2]
```diff
@@ -268,10 +268,24 @@
...
         </p>
-        <div style="min-height: 22px;"></div>
+        <div style="min-height: 8px;"></div>
+
+        <div class="usa-alert usa-alert--info usa-alert--no-icon">
+     
+  <div class="usa-alert__body">
+    <p class="usa-alert__text">
+      <a class="usa-link" href="https://www.acf.hhs.gov/ofa/policy-guidance/dear-colleague-tanf-data-portal">Read the "Dear Colleague" letter</a>
+      from Ann Flagg, Director of the Office of Family Assistance for updates on the rollout of TDP.
```
Comment (lfrohlich) [inline product-updates/knowledge-center/index.html]:
```suggestion
      from Ann Flagg, Director of the Office of Family Assistance, for key information about the rollout of TDP. 
```

**Review by lfrohlich — COMMENTED** [review]
Looks good! Two small comments

File: product-updates/knowledge-center/index.html (hunk 2 above)
Comment (ttran-hub) [reply product-updates/knowledge-center/index.html]: @reitermb Minor tag review: can we change the H2 tag to the H3 tag to conform with other question tags? other then that, a11y looks good. 🚀
image

**Comment by ttran-hub** [comment]
a11y looks good with the one tag update request; can we change the h2 to h3 to conform with other questions in the file? Aside from that, a11y is ready to go.
image

**Comment by reitermb** [comment]
> a11y looks good with the one tag update request; can we change the h2 to h3 to conform with other questions in the file? Aside from that, a11y is ready to go. image

Definitely an easy change, but I _think_ it might be preferable from an a11y standpoint to leave it as is to stay consistent with the practice of [not skipping heading levels](https://www.w3.org/WAI/tutorials/page-structure/headings/). My thinking was that while using an H2 does structurally single that item out from the other FAQs, that's actually more or less consistent with what's being communicated visually/in the organization of the page by how to provide feedback being on its own; it's own category. 

Thoughts?

**Comment by reitermb** [comment]
Can't remember if quoting sends a ping—cc: @ttran-hub @ADPennington

**Comment by ttran-hub** [comment]
> 

> Can't remember if quoting sends a ping—cc: @ttran-hub @ADPennington

I was bouncing from header to header; it seems like we missed representing the h2 and h3. Should we add a h2 tag for Bug Reporting?

**Comment by reitermb** [comment]
> > 
> 
> > Can't remember if quoting sends a ping—cc: @ttran-hub @ADPennington
> 
> I was bouncing from header to header; it seems like we missed representing the h2 and h3. Should we add a h2 tag for Bug Reporting?

I'm good with that! Added (All changes including this one are now pushed to Test as well)

**Review by lfrohlich — APPROVED** [review]
Looks good to me!

**Review by ADPennington — APPROVED** [review]
thanks @reitermb @ttran-hub!

as our KC gets longer, it will be helpful to include the section anchors/links at the top. this can help ensure there is an inherent order to FAQs for new users while also helping those more familiar with TDP get to the areas where they may need the most support. 

LGTM!

**Pull request merged**